api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -19,6 +19,18 @@
 WEBHOOK_SECRET = "secret-key"
 
 
+def mocked_requests_delete(*args, **kwargs):
+    class MockResponse:
+        def __init__(self, json_data, status_code):
+            self.json_data = json_data
+            self.status_code = status_code
+
+        def raise_for_status(self) -> None:
+            pass
+
+    return MockResponse(json_data={"data": "data"}, status_code=204)
```

Comment:
Is there a reason to keep this at module level instead of having this code inside the test?

Reply:
I don't think this is needed at all, see [this comment](https://github.com/Flagsmith/flagsmith/pull/3962/files#r1603266828).

Reply:
You're right. Removed at all in favor of the simpler solution you recommended.